### PR TITLE
fix: improve dependency graph design and layout

### PR DESCRIPTION
**Pull request by Traxmaxx**
## Problem

Bigger graphs look a little bit odd sometimes and I want to get closer to the original designs

## Solution

I changes some of the layout parameters (coent-bilkent) to get closer to the designs when rendering the graph.

Before:
Screenshot 2023-09-29 at 15 26 44

after:
Screenshot 2023-09-29 at 15 33 53

## Changes Made

See diff

## How to Test

[Provide instructions on how to test the changes you made, including any relevant details like configuration steps or data to be used for testing.]

## Screenshots

[Include screenshots, if relevant, to help reviewers understand the changes you made.]

## Notes

[Any additional notes or information that you would like to share with the reviewers.]

## Checklist

- [x] Code follows the <a href="https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md">contributing</a> guidelines
- [x] Changes have been thoroughly tested
- [x] <a href="https://github.com/tailwarden/docs.komiser.io">Documentation</a> has been updated, if necessary
- [x] Any dependencies have been added to the project, if necessary

## Reviewers

@[username of the reviewer]

### CODE REVIEW

**Review by AvineshTripathi — APPROVED** [review]
LGTM

**Review by shivam-sharma7 — APPROVED** [review]

File: dashboard/components/explorer/DependencyGraph.tsx [hunk 1]
```diff
@@ -146,8 +148,14 @@ const DependencyGraph = ({ data }: DependencyGraphProps) => {
         ]}
         cy={(cy: Cytoscape.Core) => cyActionHandlers(cy)}
       />
-      <div className="absolute bottom-0 left-0 bg-black-100 text-black-400">
+      <div className="absolute bottom-0 left-0 flex gap-2 overflow-visible overflow-visible bg-black-100 text-black-400">
```
Comment (shivam-sharma7) [inline dashboard/components/explorer/DependencyGraph.tsx:151]:
```suggestion
      <div className="absolute bottom-0 left-0 flex gap-2 overflow-visible bg-black-100 text-black-400">
```

Comment (shivam-sharma7) [reply dashboard/components/explorer/DependencyGraph.tsx:151]: keep one

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]

**Comment by Traxmaxx** [comment]
I added some more improvements based on feedback from @AllieMendes and @jakepage91 (thanks a lot 🙏)

Show full name on hover and also show relation type on hovering edges. Additionally we now show the total resource count in the bottom left corner and there's a tooltip which explains we just display AWS infrastructure for the interim period.
Screenshot 2023-10-02 at 12 00 13
Screenshot 2023-10-02 at 12 00 17
Screenshot 2023-10-02 at 12 00 25

**Comment by shivam-sharma7** [comment]
@Traxmaxx This is amazing

**Review by shivam-sharma7 — APPROVED** [review]

**Review by mlabouardy — APPROVED** [review]

**Pull request merged**